keras_hub/src/models/retinanet/retinanet_label_encoder_test.py

```diff
@@ -1,26 +1,33 @@
 import numpy as np
 from keras import ops
 
+from keras_hub.src.models.retinanet.anchor_generator import AnchorGenerator
 from keras_hub.src.models.retinanet.retinanet_label_encoder import (
     RetinaNetLabelEncoder,
 )
 from keras_hub.src.tests.test_case import TestCase
 
 
 class RetinaNetLabelEncoderTest(TestCase):
+    def setUp(self):
+        self.anchor_generator = AnchorGenerator(
+            bounding_box_format="xyxy",
+            min_level=3,
+            max_level=7,
+            num_scales=3,
+            aspect_ratios=[0.5, 1.0, 2.0],
+            anchor_size=8,
+        )
+
     def test_layer_behaviors(self):
         images_shape = (8, 128, 128, 3)
         boxes_shape = (8, 10, 4)
         classes_shape = (8, 10)
         self.run_layer_test(
             cls=RetinaNetLabelEncoder,
             init_kwargs={
```

Comment:
define the init_kawrgs in setup. refer other layer tests for example

Reply:
Done